scripts/extension-version.test.js

```diff
@@ -0,0 +1,149 @@
+const { expect } = require('chai')
+
+const { nextExtensionVersion } = require('./extension-version')
+
+describe('next extension version', () => {
+  it('it should work with stable channel and current version to new version schema', () => {
```

Comment:
I don't really understand what you're trying to test here? (more specifically, what's the `new version schema`?)

Reply:
The current versioning schema is patch based, defined in this issue https://github.com/prisma/vscode/issues/114

The new versioning schema is defined in this issue https://github.com/prisma/vscode/issues/121#issuecomment-620643356

Please let me know if that makes it clear?

Reply:
ah okay, understood